### PR TITLE
Improve the component styling

### LINKED ISSUE #3: Improve the component stying by adding positive / negative / zero dynamic class
The component should have:
- a `money-amount--positive` class if the amount is positive
- a `money-amount--negative` class if the amount is negative
- a `money-amount--zero` class if the amount is exactly 0

### PR DESCRIPTION
**Pull request by meliecool**
resolves #3

### CODE REVIEW

File: .storybook/main.js [hunk 1]
```diff
@@ -19,6 +19,20 @@ module.exports = {
...
 
+    config.module.rules.push(
+      {
+        test: /\.scss$/,
+        use: [
+          { loader: 'style-loader' },
+          { loader: 'css-loader' },
+          {
+            loader: 'sass-loader',
+          },
```
Comment (lunatetzuka) [inline .storybook/main.js:30]:
```suggestion
          { loader: 'sass-loader' },
```

File: package.json [hunk 2]
```diff
@@ -21,12 +21,16 @@
     "build:storybook": "build-storybook -c .storybook -o dist/storybook"
   },
   "dependencies": {
+    "css-loader": "^5.0.0",
```
Comment (lunatetzuka) [inline package.json:24]: move to devDependencies

File: package.json [hunk 3]
```diff
@@ -21,12 +21,16 @@
...
     "get-user-locale": "1.4.0",
     "iso4217": "0.2.0",
+    "sass-loader": "^10.0.4",
```
Comment (lunatetzuka) [inline package.json:27]: move to devDependencies

File: package.json [hunk 4]
```diff
@@ -21,12 +21,16 @@
...
+    "style-loader": "^2.0.0",
     "vue": "^2.x"
   },
   "devDependencies": {
     "@babel/preset-env": "^7.12.1",
+    "@storybook/addon-actions": "^5.3.19",
```
Comment (lunatetzuka) [inline package.json:33]: to remove ?

File: src/MoneyAmount.vue [hunk 5]
```diff
@@ -33,8 +33,14 @@ export default {
...
-    classes() {
-      return 'money-amount';
+    classes () {
+      if (this.amount > 0) {
+        return 'money-amount--positive'
+      } else if (this.amount < 0) {
+        return 'money-amount--negative'
+      } else {
+        return 'money-amount--zero'
+      }
```
Comment (lunatetzuka) [inline src/MoneyAmount.vue:43]:
```suggestion
     return this.amount > 0
       ? 'money-amount--positive'
       : this.amount < 0
         ? 'money-amount--negative'
         : 'money-amount--zero';
```

**Comment by meliecool** [comment]
Ok! I add the colors optional + change dependencies to dev

File: src/MoneyAmount.vue (hunk 5 above)
Comment (meliecool) [reply src/MoneyAmount.vue]: The lint is too severe for that

**Review by LukaszKokot — CHANGES_REQUESTED** [review]
I don't want any colors by default, as the CSS will inevitably vary from a project to another. I would just like the CSS classes to be added though.
So having `money-amount--positive` or `money-amount--negative` or `money-amount--zero` is perfect, but no styling please.
If you want, you can however configure your storybook stories so that the amounts are styled, to demonstrate that it works!

File: src/MoneyAmount.vue [hunk 6]
```diff
@@ -31,10 +31,19 @@ export default {
       required: false,
       default: 0,
     },
+    hasColors: {
```
Comment (LukaszKokot) [inline src/MoneyAmount.vue:34]: Please remove this property.

File: src/MoneyAmount.vue [hunk 7]
```diff
@@ -48,3 +57,14 @@ export default {
   },
 };
 </script>
+<style lang="scss">
```
Comment (LukaszKokot) [inline src/MoneyAmount.vue:60]: Please remove the styling

File: src/MoneyAmount.vue (hunk 6 above)
Comment (meliecool) [reply src/MoneyAmount.vue]: It's done

File: src/MoneyAmount.vue (hunk 7 above)
Comment (meliecool) [reply src/MoneyAmount.vue]: It's done too

File: .storybook/preview-body.html [hunk 8]
```diff
@@ -0,0 +1,9 @@
+<style>
+  .money-amount--positive {
+    color: green;
+  }
+
+  .money-amountnegative {
```
Comment (lunatetzuka) [inline .storybook/preview-body.html:6]:
```suggestion
  .money-amount--negative {
```

**Review by LukaszKokot — APPROVED** [review]
If you fix @lunatetzuka  's comment, it's good!

**Review by lunatetzuka — APPROVED** [review]

**Pull request merged**